src/player.h

```diff
@@ -238,6 +243,13 @@ namespace Player {
 	/** Game title. */
 	extern std::string game_title;
 
+	/**
+	 * The default speed modifier applied when the speed up button is pressed
+	 *  Only used for configuring the speedup, don't read this var directly use
+	 *  GetSpeedModifier() instead.
+	 */
+	extern int speed_modifier;
```

Comment:
The idea was to have an exported variable that a potential setting dialog could alter to get more/less speedup. But currently its useless :D